atom/browser/atom_browser_client.cc

```diff
@@ -322,6 +323,11 @@ void AtomBrowserClient::AppendExtraCommandLineSwitches(
         web_contents, command_line);
     SessionPreferences::AppendExtraCommandLineSwitches(
         web_contents->GetBrowserContext(), command_line);
+
+    auto context_id = atom::api::WebContents::GetIDForContents(
+      web_contents);
```

Comment:
Indentation here doesn't look right.
Did `clang-format` throw any warnings?

Reply:
@alexeykuzmin I think this was before clang-format was introduced